refinery/tool_manager/tests.py

```diff
@@ -2518,19 +2519,22 @@ def _start_visualization(
             post_response = self.tools_view(self.post_request)
             self.assertEqual(post_response.status_code, 200)
 
-            tool = VisualizationTool.objects.get(
+            tools = VisualizationTool.objects.filter(
                 tool_definition__uuid=self.td.uuid
             )
-            self.assertEqual(tool.get_owner(), self.user)
+            if count:
+                self.assertEqual(len(tools), count)
+            last_tool = tools[len(tools)-1]
```

Comment:
Shorthand for this would be: `tools[:-1]`

Reply:
not quite:
```
>>> a = [1,2,3]
>>> a[:-1]
[1, 2]
>>> a[-1:]
[3]
>>> a[-1:][0]
3
>>> a[-1]
3
```
But [alas](https://docs.djangoproject.com/en/1.11/topics/db/queries/#limiting-querysets):
> Negative indexing (i.e. Entry.objects.all()[-1]) is not supported

Reply:
[last()](https://docs.djangoproject.com/en/1.7/ref/models/querysets/#last) returns the last object in the queryset

Reply:
@mccalluc Oops, my mistake w/ the shorthand comment.

Regarding the `alas` though, one could do: `Entry.objects.last()`